### ISSUE TITLE
Favicon?

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/grackl#101]
do we need one?

**Comment by toribr** [comment]
Yes, and I have some direction from what to do from Diana. Will get it together shortly.

**Comment by toribr** [comment]
[favicon.zip](https://github.com/cityofaustin/grackl/files/1543859/favicon.zip)
